### PR TITLE
Fix for boundary event graphicalinfo calculation.

### PR DESCRIPTION
**Pull request by ddpardo**
When converting an (java) bpmnmodel to json the boundary event must take the parents graphicalinfo into account.

### CODE REVIEW

**Comment by dbmalkovsky** [comment]
Thanks for the contribution.

Could you please add a license header to ` BoundaryEventGraphicInfoTest`?  Thanks.

A copy can be found at `flowable-engine/ide-settings/idea/fileTemplates/includes/Copyright.java`

**Comment by ddpardo** [comment]
I have updated the test class with the copyright information.

**Comment by ddpardo** [comment]
Could you provide an update on this mergerequest?

**Comment by ddpardo** [comment]
Any feedback on this?

File: modules/flowable-json-converter/src/main/java/org/flowable/editor/language/json/converter/BoundaryEventJsonConverter.java [hunk 1]
```diff
@@ -87,9 +89,24 @@ protected void convertElementToJson(ObjectNode propertiesNode, BaseElement baseE
         ObjectNode dockNode = objectMapper.createObjectNode();
         GraphicInfo graphicInfo = model.getGraphicInfo(boundaryEvent.getId());
         GraphicInfo parentGraphicInfo = model.getGraphicInfo(boundaryEvent.getAttachedToRef().getId());
-        dockNode.put(EDITOR_BOUNDS_X, graphicInfo.getX() - parentGraphicInfo.getX());
-        dockNode.put(EDITOR_BOUNDS_Y, graphicInfo.getY() - parentGraphicInfo.getY());
+        BigDecimal parentX = new BigDecimal(parentGraphicInfo.getX());
```
Comment (martin-grofcik) [inline modules/flowable-json-converter/src/main/java/org/flowable/editor/language/json/converter/BoundaryEventJsonConverter.java]: the same logic should be used in org.flowable.cmmn.editor.json.converter.CriterionJsonConverter#convertElementToJson
Are you going to add it there?

**Pull request merged**